packages/chat/components/chat/src/chat.scss

```diff
@@ -0,0 +1,274 @@
+/**
+ * Copyright IBM Corp. 2023, 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@use '../../../../../globals/scss/vars' as *;
+
+@use '@carbon/styles/scss/theme' as *;
+
+:host(#{$c4ai-prefix}-chat) {
+  position: relative;
+  display: block;
+  block-size: 320px;
+  font-family: 'IBM Plex Sans', sans-serif;
+  min-block-size: 320px;
+
+  .#{$c4ai-prefix}--chat-container {
+    padding: 20px 20px 20px 35px;
+    border-radius: 10px;
+    background: #1c1c17;
+    block-size: 400px;
+    inline-size: 100%;
+    max-inline-size: 500px;
+    min-block-size: 300px;
+  }
+
+  .chat-messages {
```

Comment:
all of the classes should be in this similar format`.#{$c4ai-prefix}--chat-container`, so for `.chat-messages`, `.user-message` and so on they should look like `.#{$c4ai-prefix}--chat-messages`